extmod/moducryptolib.c

```diff
@@ -82,6 +94,19 @@ typedef struct _mp_obj_aes_t {
     uint8_t key_type: 2;
 } mp_obj_aes_t;
 
+STATIC inline bool is_ctr_mode(int block_mode) {
+#if MICROPY_PY_UCRYPTOLIB_CTR
```

Comment:
Please use indented directives, as majority of the rest of code does.

Reply:
Done